public/js/src/module/controller-webform.js

```diff
@@ -14,6 +14,7 @@ import records from './records-queue';
 import $ from 'jquery';
 import encryptor from './encryptor';
 import formCache from './form-cache';
+import { getLastSavedRecord, populateLastSavedInstances } from './last-saved';
```

Comment:
I tried to keep changes here as minimal as possible, as this module is still untested. The changes here are to support:

- The signature change to `uploadRecord`
- Continuing to ensure last-saved instances are populated when the form is reset after submitting/queueing/saving a draft record